atomic_reactor/plugins/post_koji_import.py

```diff
@@ -722,12 +779,22 @@ class KojiImportSourceContainerPlugin(KojiImportBase):
 
     key = PLUGIN_KOJI_IMPORT_SOURCE_CONTAINER_PLUGIN_KEY  # type: ignore
 
-    def get_output(self, buildroot_id):
+    def get_output(self, buildroot_id: str) -> List[Dict[str, Any]]:
+        outputs = super().get_output(buildroot_id)
         pullspec = get_unique_images(self.workflow)[0]
-
-        return koji_get_output(workflow=self.workflow, buildroot_id=buildroot_id,
-                               pullspec=pullspec, platform=os.uname()[4],
-                               source_build=True)
+        metadatas, output_file = koji_get_output(
+            workflow=self.workflow,
+            buildroot_id=buildroot_id,
+            pullspec=pullspec,
+            platform=os.uname()[4],
+            source_build=True,
+        )
+        self.workflow.data.koji_upload_files.append({
+            "local_filename": output_file.filename,
+            "dest_filename": metadatas[0]['filename'],
+        })
```

Comment:
Hmm, so the `output_file` returned from `koji_get_output()` has to correspond to `metadatas[0]`?

Reply:
This is what I understand. For source build, the first element of the return value is a list of metadata, only one there, and the second element is the Output object including the file.

Reply:
I feel like this relies on the internal logic of `koji_get_output()` too much. Could the dest_filename be `output_file.metadata['filename']`?

Reply:
Updated.